pom.xml

```diff
@@ -58,6 +58,11 @@
       <artifactId>httpmime</artifactId>
       <version>4.3.3</version>
     </dependency>
+    <dependency>
+    	<groupId>commons-collections</groupId>
+    	<artifactId>commons-collections</artifactId>
+    	<version>3.2.1</version>
+    </dependency>
```

Comment:
What is the motivation behind adding these plugins? Did something not work before?